pkg/restore/crd_v1_preserve_unknown_fields_action.go

```diff
@@ -74,6 +79,11 @@ func (c *CRDV1PreserveUnknownFieldsAction) Execute(input *velero.RestoreItemActi
 
 		// Make sure all versions are set to preserve unknown fields
 		for _, v := range crd.Spec.Versions {
+			// If the Schema is nil, there are no nested fields to set, so skip over it for this version.
+			if v.Schema == nil {
+				continue
+			}
+
 			// Use the address, since the XPreserveUnknownFields value is undefined or true (false is not allowed)
 			preserve := true
 			v.Schema.OpenAPIV3Schema.XPreserveUnknownFields = &preserve
```

Comment:
TODO: `Schema.OpenAPIV3Schema` could also be a nil pointer, so check it before accessing, too.

Reply:
The `Schema.OpenAPIV3Schema.XPreserveUnknownFields` could also be a nil pointer.

Reply:
do we need to add a nil-check for `v.Schema.OpenAPIV3Schema`?

Reply:
Ah, yeah, thanks!

Reply:
@jenting I think the `XPreserveUnknownFields` nil pointer is fine, since we're setting it here explicitly.